modular_skyrat/modules/events/code/event_overrides.dm

```diff
@@ -96,7 +96,8 @@
  * Does not have policy. Will re-add if/when policy is added
  */
 /datum/round_event_control/operative
-	max_occurrences = 0
+	//max_occurrences = 0
```

Comment:
why comment out this line?

Reply:
Commenting out is better, in case the file is updated. Less compatibility headaches.